types/index.d.ts

```diff
@@ -365,7 +365,11 @@ declare namespace math {
     version: string;
 
     expression: MathNode;
-    json: MathJsJson;
+
+    /**
+     * Returns reviver function that can be used as reviver in JSON.parse function.
+     */
+     reviver(): (key: any, value: any) => any;
 
```

Comment:
Just a question: why does this need to be moved out here, from inside the MathJsJson interface? What went wrong in the previous structure? And is that situation now tested in types/index.ts?